.circleci/config.yml

```diff
@@ -295,8 +301,43 @@ jobs:
           key: v1-ipfs-webui-{{ checksum "~/ipfs/go-ipfs/ipfs-webui/package-lock.json" }}
           paths:
             - ~/ipfs/go-ipfs/ipfs-webui/node_modules
+  docker-build:
+    executor: dockerizer
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Build Docker image
+          command: |
+            docker build -t $IMAGE_NAME:$WIP_IMAGE_TAG .
+      - run:
+          name: Archive Docker image
+          command: docker save -o go-ipfs-image.tar $IMAGE_NAME
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./go-ipfs-image.tar
+  docker-push:
+    executor: dockerizer
+    steps:
+      - checkout
+      - setup_remote_docker
+      - attach_workspace:
+          at: /tmp/workspace
+      - run:
+          name: Load archived Docker image
+          command: docker load -i /tmp/workspace/go-ipfs-image.tar
+      - run:
+          name: Publish Docker Image to Docker Hub
+          command: |
+            echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
```

Comment:
When are these credentials provided?

Reply:
Every branch, it looks like? That's not a good idea.

Reply:
I keep running into this same wall. 🤦‍♂

Reply:
Resolution: only users in a specific group will get this token. That will _also_ be the group with release permissions.